vendor.go

```diff
@@ -33,8 +33,24 @@ type ListVendorResponse struct {
 
 // ListVendorOptions is the data structure used when calling the ListVendors API endpoint.
 type ListVendorOptions struct {
-	APIListObject
-	Query string `url:"query,omitempty"`
```

Comment:
`query` does not appear in the API documentation for this call. So I removed it.